src/Dfc.CourseDirectory.Functions/Dfc.CourseDirectory.Functions.csproj

```diff
@@ -6,10 +6,13 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Azure.Search.Documents" Version="11.1.1" />
+    <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="8.0.0" />
```

Comment:
Shouldn't need this reference as it should be inherited from `Core`?

Reply:
yep, thanks, done